adapta/storage/models/local.py

```diff
@@ -27,6 +27,13 @@ class LocalPath(DataPath):
     Local file system path.
     """
 
+    @classmethod
+    def from_uri(cls, url: str) -> "DataPath":
+        raise NotImplementedError
+
+    def to_uri(self) -> "DataPath":
+        raise NotImplementedError
```

Comment:
I guess the local path is easy to covnert to/from URIs with the os.path lib?

Reply:
yeah that would be the same as returning `path` property though? i'll look into it